Gemfile.lock

```diff
@@ -322,10 +321,12 @@ DEPENDENCIES
   simplecov-rcov (~> 0.2.3)
   slimmer (= 8.2.1)
   smartdown (~> 0.15.1)
-  syck (= 1.0.5)
   therubyracer (~> 0.12.1)
   tilt (= 1.4.1)
   timecop
   uglifier
   unicorn (= 4.8.3)
   webmock (= 1.20.4)
+
+BUNDLED WITH
+   1.10.5
```

Comment:
I'm using rbenv with rbenv-default-gems plugin that installs me bundler and some other gems automatically when new ruby versions are installed.

Ok, I'll downgrade and remove this change :)